taxonomies/food/ingredients.txt

```diff
@@ -176,21 +176,10 @@ stopwords:zh: :配料
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
 en: imazalil
-da: imazalil
-de: Imazalil
```

Comment:
German capitalises all nouns, so if the `xx:` entry isn’t capitalised (and it probably shouldn’t be in this case), the `de:` entry should still be there so it won’t show up lower-cased in German locales.